swfiles/+swplot/logo.m

```diff
@@ -100,9 +100,9 @@ function logo(varargin)
     
     txt0 = sprintf([ver0 '\nWritten by:\n  S' char(225) 'ndor T' char(243) ...
         'th\n  sandor.toth@psi.ch\n  Paul Scherrer Institut\n\nContributed:\n  Simon Ward\n  Mechthild Enderle\n  Bj' char(246) 'rn F' ...
-        char(229) 'k\n  Duc Manh Lee\n\n' ...
+        char(229) 'k\n  Duc Manh Le\n\n' ...
```

Comment:
Thanks for this. You should probably add your name here too!

Reply:
Good point!